components/ActionSheet/src/private/MDCActionSheetHeaderView.h

```diff
@@ -34,4 +34,12 @@
 
 @property (nonatomic, strong, nonnull) UIFont *messageFont;
 
+@property(nonatomic, strong, nullable) UIColor *primaryColor;
```

Comment:
Please remove `primaryColor` and `secondaryColor`.

Reply:
Primary color is required for a themer.

Reply:
I believe the themer can simply set the titleTextColor and messageTextColor, correct? Why are separate properties required just for a themer?

Reply:
If there is only one label then that will be `onSurfaceColor` with `0.6` alpha. If there is both then the title will be `0.87` alpha. It depends if there is both labels or not and if a client applies a themer and then sets a label the theme won't be applied. `titleTextColor` and `messageTextColor` are not required, `primary` and `secondary` are.